app/views/helpers/siteHeader.scala.html

```diff
@@ -21,7 +21,7 @@
 @(showSignOut: Boolean = true)(implicit messages: Messages)
 
 @hmrcHeader(Header(
```

Comment:
We could potentially upgrade to use HmrcStandardHeader within the govuk_wrapper file and get rid of this siteHeader. I think it should give us more out of the box instead of specifying the gov uk url and containerClasses for example. E.g.
headerBlock = hmrcHeader(
        serviceUrl  = Some(controllers.routes.SoftwareChoicesController.show.url),
        serviceName = Some(messages("common.service.name")),
        signOutUrl  = None,
        phaseBanner = None
    ),

Reply:
That's better approach, done. Thanks.